db/dev_seeds/polls.rb

```diff
@@ -28,19 +28,49 @@
               ends_at:   1.month.ago,
               results_enabled: true,
               stats_enabled: true)
+
+  Poll.all.each do |poll, i|
```

Comment:
You are right, I think I wanted to use it at first but at the end, I didn't and I forgot to remove it. Thanks!

Reply:
@microweb10 I'd also like to add that you should use `Poll.find_each` instead of `Poll.all.each` as the latter can exceed the amount of memory available if said model is associated with a table with a considerable amount of records, as mentioned [here](https://guides.rubyonrails.org/v4.2/active_record_querying.html#retrieving-multiple-objects-in-batches)

Reply:
resolved https://github.com/consul/consul/pull/2833/commits/dc01291ad26ae5189f421a4a915ab9a6353f611a